public/src/components/ButtonGroup.jsx

```diff
@@ -1,23 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 const StyledButtonGroup = styled.div`
+  
+ ${({ horizontal, theme }) => (horizontal ? css`
+    button + button {
+      border-radius: 0;
+    }
+    label  + label {
+      border-radius: 0;
+    }
+    button:first-child {
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+    }
+    label:first-child {
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+    }
+    & > button:last-of-type {
+      border-top-right-radius: ${theme.borderRadius};
+      border-bottom-right-radius: ${theme.borderRadius};
+    }
+    & > label:last-of-type {
```

Comment:
I believe you could do:

```
& > label, & > button {
  &: last-of-type {
    ...
  }
}
```

if not, you could just do:
```
& > button:last-of-type, & > label:last-of-type {}
```

also, why are labels having border radii?

Reply:
Cuz the label is surrounding a input checkbox, which I've made to look like a button xD

Reply:
![image](https://user-images.githubusercontent.com/9252682/108531794-199fb780-72a5-11eb-8dea-c4c09546f187.png)

Reply:
Those are checkboxes (yes they should be radio buttons technically, but I got lazy)

Reply:
Originally wrote that component for this: 
![image](https://user-images.githubusercontent.com/9252682/108531888-389e4980-72a5-11eb-8424-f699e227d8f9.png)